modules/Matting/AlphaGAN-Matting/AlphaGAN/alphaGAN_eval.py

```diff
@@ -0,0 +1,241 @@
+import torchvision.transforms as transforms
+import numpy as np
+import torch as t
+from model.AlphaGAN import NetG
+from PIL import Image
+import os
+from visualize import Visualizer
+import tqdm
+
+os.environ["CUDA_VISIBLE_DEVICES"] = '2'
+vis = Visualizer('alphaGAN_eval')
+
+transform = transforms.Compose([
+            transforms.ToTensor(),
+            transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+        ])
+
+to_pil = transforms.Compose([
+    transforms.ToPILImage()
+])
+
+MODEL_DIR = '/data1/zzl/model/alphaGAN/new_trainset/netG/netG_40.pth'
+
+
+def padding_img(img):
+
+    img_size = np.shape(img)
+    if len(img_size) == 3:
+        (h, w, c) = img_size
+        w_padding = (int(w/320) + 1) * 320
+        h_padding = (int(h/320) + 1) * 320
+
+        padding_result = np.pad(img, ((0, h_padding - h), (0, w_padding - w), (0, 0)), 'mean')
+
+        return Image.fromarray(padding_result), int(w_padding/320), int(h_padding/320)
+    elif len(img_size) == 2:
+        (h, w) = img_size
+        w_padding = (int(w/320) + 1) * 320
+        h_padding = (int(h/320) + 1) * 320
+
+        padding_result = np.pad(img, ((0, h_padding - h), (0, w_padding - w)), 'constant', constant_values=0)
+
+        return Image.fromarray(padding_result), int(w_padding/320), int(h_padding/320)
+    else:
```

Comment:
This `else` isn't needed. And by the comment of @StevenPuttemans may be the whole branching can be eliminated? Am I right?